editor/plugins/texture_region_editor_plugin.h

```diff
@@ -66,45 +67,47 @@ class TextureRegionEditor : public AcceptDialog {
 	SpinBox *sb_off_x = nullptr;
 	SpinBox *sb_sep_y = nullptr;
 	SpinBox *sb_sep_x = nullptr;
-	Panel *edit_draw = nullptr;
+
+	PanelContainer *texture_preview = nullptr;
+	Panel *texture_overlay = nullptr;
 
 	VScrollBar *vscroll = nullptr;
 	HScrollBar *hscroll = nullptr;
 
 	Vector2 draw_ofs;
-	float draw_zoom = 0.0;
+	float draw_zoom = 1.0;
 	bool updating_scroll = false;
 
-	int snap_mode = 0;
+	int snap_mode = SnapMode::SNAP_NONE;
```

Comment:
Couldn't it be:
```suggestion
	SnapMode snap_mode = SNAP_NONE;
```
?